Jenkinsfile

```diff
@@ -25,9 +25,8 @@ pipeline {
         stage('Prepare') {
             steps {
                 script {
-                    if (env.BRANCH_NAME =~ /^PR-\d+$/) {
-                        def pr_number = sh (script: "echo ${env.BRANCH_NAME} | grep -o -E '[0-9]+' ", returnStdout: true)
-                        def json = sh (script: "curl -s https://api.github.com/repos/crossplaneio/crossplane/pulls/${pr_number}", returnStdout: true).trim()
+                    if (env.CHANGE_ID == null) {
```

Comment:
i would have expected this to be `env.CHANGE_ID != null`, meaning that we have a change_id value and can therefore use it in the curl call on the next line.  what am i missing?

Reply:
That is correct, good catch.